### PR TITLE
Add missing transit to API.

### PR DESCRIPTION
**Pull request by laurentg**
Here we are! Comments welcomed.

### CODE REVIEW

**Comment by branflake2267** [comment]
Wow, I didn't expect that. :) Nice job.

File: gwt-maps-showcase/src/main/java/com/google/gwt/maps/testing/client/maps/TransitDirectionsServiceMapWidget.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
+package com.google.gwt.maps.testing.client.maps;
```
Comment (branflake2267) [inline gwt-maps-showcase/src/main/java/com/google/gwt/maps/testing/client/maps/TransitDirectionsServiceMapWidget.java:1]: Nice :)

**Comment by twistedpair** [comment]
@laurentg thanks! I did not see any test cases to cover this. Could you add those too?

**Comment by branflake2267** [comment]
I don't know if you'd be up to adding some test cases. :) But those would be welcome :) lol @twistedpair  beat me to it. :)

**Comment by laurentg** [comment]
Thanks for the comments. I'll prepare some test-case, stay tuned for more commits...

**Comment by laurentg** [comment]
I checked the existing test cases and added a new one, but how can I run it only without all the others?

**Comment by branflake2267** [comment]
Right click on the test case and debug as GWT Test case? The project will take a few moments and start compiling and then test the case.

**Comment by branflake2267** [comment]
IDEA its right click and debug as test.

**Comment by laurentg** [comment]
Unfortunately, I'm on Eclipse and I don't have the right option. However I found a workaround (run as JUnit test, then add GWT library and source folders to run configuration). I will commit my unit test tomorrow.

**Comment by laurentg** [comment]
Read: "I will commit my unit test today". Time to go to bed :) Btw, which code formatting do you use? I'd like to have auto-format settings for my Eclipse, it would be easier.

**Comment by branflake2267** [comment]
I use the same format as the gwt project, except I go 120 characters for width. 2 space indent...

**Comment by branflake2267** [comment]
Nice work

**Comment by branflake2267** [comment]
More specific, I create a Eclipse custom formatter and I only set the tabs to spaces and I make them 2. Then I change the line length to 120. The rest I leave to default.

**Comment by twistedpair** [comment]
@branflake2267 New project team I started on has an export of that format file on their wiki. Perhaps we can do the same? I think there are some maven plugins that can also automatically apply that too.

@laurentg Thanks for working on the tests. Right clicking on the project > Google > Web Toolkit Application, and setting this application as a GWT application should enable the GWT Eclipse plugin integration for you. This will add options like "RunAs > GWT Unit Test" for you.

**Comment by branflake2267** [comment]
Yeah, thats a great idea. Its easy to export the xml format. We could put that in the repo.

**Comment by laurentg** [comment]
@twistedpair Thanks for the tip, indeed maven did not created my project with GWT enabled. When enabled, I have the "Debug as > GWT Unit test" option appearing, and this works fine. Good to know and simpler to setup than my hack...

@branflake2267 I setup the code formatter according to specs. Indeed a xml export of the formatter in the project could be fine.

**Comment by branflake2267** [comment]
Nice work and great job.

**Pull request merged**